### PR TITLE
Improvements to PR and Issue templates

### PR DESCRIPTION
**Pull request by mstanleyjones**
### Describe the proposed changes

Took the PTAL section out of the PR template since you can assign reviewers now
A few other small improvements

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview ready!

Built with commit d33c466acb3437fd9e4faad21ed00f1d8ecddcb5

https://deploy-preview-829--docker-docs.netlify.com

**Comment by londoncalling** [comment]
LGTM, beat me to it :-)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
+    as you go.
 
-<!--DO NOT edit files and directories listed in .NOT_EDITED_HERE.yaml.
-    These are maintained in upstream repos and changes here will be lost.-->
+    DO NOT edit files and directories listed in .NOT_EDITED_HERE.yaml.
+    These are maintained in upstream repos and changes here will be lost.
+
+    Details are appreciated and labels / milestones / reviewers are helpful -->
 
 ### Describe the proposed changes
```
Comment (londoncalling) [inline .github/PULL_REQUEST_TEMPLATE.md]: How about just "Proposed Changes" so we have to edit less, and it's the same format as the other headings?

**Comment by mstanleyjones** [comment]
Good idea, @londoncalling . I made that change. PTAL.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,29 +1,23 @@
 <!--Thanks for your contribution. See [CONTRIBUTING](CONTRIBUTING.md)
-    for this project's contribution guidelines. -->
+    for this project's contribution guidelines. Remove these comments
+    as you go.
 
-<!--DO NOT edit files and directories listed in .NOT_EDITED_HERE.yaml.
-    These are maintained in upstream repos and changes here will be lost.-->
+    DO NOT edit files and directories listed in .NOT_EDITED_HERE.yaml.
```
Comment (johndmulhausen) [inline .github/PULL_REQUEST_TEMPLATE.md]: `_data/not_edited_here.yaml`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,29 +1,23 @@
...
+    These are maintained in upstream repos and changes here will be lost.
 
-### Describe the proposed changes
+    Details are appreciated and labels / milestones / reviewers are helpful -->
```
Comment (johndmulhausen) [inline .github/PULL_REQUEST_TEMPLATE.md]: s/are helpful/will help get this issue addressed more quickly./ 
(Or somesuch)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,29 +1,23 @@
...
-
-<!-- Refer to related PRs or issues: #1234, or 'Fixes #1234' or 'Closes #1234'. -->
+<!--Tell us what you did and why.-->
 
-### Related issue or PR in another project
-
-<!-- Full URLs to issues or pull requests in other Github projects -->
-
-### Please take a look
+### Unreleased project version
```
Comment (johndmulhausen) [inline .github/PULL_REQUEST_TEMPLATE.md]: Add "(optional)" to the end

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -1,29 +1,23 @@
...
 
-<!-- At-mention specific individuals or groups, like @exampleuser123 -->
+<!--If this change only applies to an unreleased version of a project, note
+    that here and base your work on the `vnext-` branch for your project.
+    Set a milestone if appropriate. -->
 
+### Related issues
```
Comment (johndmulhausen) [inline .github/PULL_REQUEST_TEMPLATE.md]: Add "(optional)" to the end

Comment (mstanleyjones) [inline .github/PULL_REQUEST_TEMPLATE.md]: Addressed your comments.

**Comment by mstanleyjones** [comment]
@johndmulhausen ball is in your court.

**Pull request merged**